src/convertToObject.js

```diff
@@ -6,7 +6,40 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const resObj = {};
+  const cleanedSourceString = sourceString.replace(/\t+/, ' ').trim();
```

Comment:
[CODE KNOWLEDGE] - The `replace` method is used incorrectly here. It replaces only the first occurrence because the regular expression is not global. You should use a global regular expression to replace all occurrences of tabs with a single space. For example: `sourceString.replace(/\t+/g, ' ')`.